drivers/ukvirtio/bus/Config.uk

```diff
@@ -0,0 +1,10 @@
+config VIRTIO_BUS
```

Comment:
So `virtio_bus` and `virtio_ring` are required for all Virtio devices, and it makes no sense to select them without selecting a device. Therefore we can hide them from `menuconfig` altogether by making them invisible options:
```
config VIRTIO_BUS
      bool
      default n
      select LIBUKBUS
      select LIBUKALLOC
      select LIBUKSGLIST
      select VIRTIO
```

Moreover, since we need both `virtio_bus` and `virtio_ring`, it makes sense to have config options of any Virtio device to enable these together.  We can do that by introducing yet another invisible variable, `CONFIG_VIRTIO` that is selected by all Virtio devices (see the earlier comments on `virtio_9p` and `virtio_blk`).

To do that, you should create `drivers/virtio/Config.uk` with:
```
+config VIRTIO
+       bool
```